Tests/SafeDIMacrosTests/InstantiableMacroTests.swift

```diff
@@ -387,50 +642,116 @@ import SafeDICore
                 """
                 public struct ExampleService: Instantiable {
                     let instantiatedA: InstantiatedA
-
-                    let nonInjectedProperty: Int
-
-                    public init(instantiatedA: InstantiatedA, nonInjectedProperty: Int = 5) {
-                        self.instantiatedA = instantiatedA
-                        self.nonInjectedProperty = nonInjectedProperty
-                    }
+
+                    let nonInjectedProperty: Int
+
+                    public init(instantiatedA: InstantiatedA, nonInjectedProperty: Int = 5) {
+                        self.instantiatedA = instantiatedA
+                        self.nonInjectedProperty = nonInjectedProperty
+                    }
+                }
+                """
+            }
+        }
+
+        func test_declaration_fixit_generatesRequiredInitializerWithDependencies() {
+            assertMacro {
+                """
+                @Instantiable
+                public struct ExampleService: Instantiable {
+                    @Instantiated
+                    let instantiatedA: InstantiatedA
+                }
+                """
+            } diagnostics: {
+                """
+                @Instantiable
+                public struct ExampleService: Instantiable {
+                                                           ╰─ 🛑 @Instantiable-decorated type must have a `public` or `open` initializer with a parameter for each @Instantiated, @Received, or @Forwarded-decorated property.
+                                                              ✏️ Add required initializer
+                    @Instantiated
+                    let instantiatedA: InstantiatedA
+                }
+                """
+            } fixes: {
+                """
+                @Instantiable
+                public struct ExampleService: Instantiable {
+                public init(instantiatedA: InstantiatedA) {
+                self.instantiatedA = instantiatedA
+                }
```

Comment:
Expected that indentation on a lot of these fixits looks off?

Reply:
yeah – Xcode corrects it. I explicitly don't want to manage indentation in fixits because I don't know the end-user's preferences.